src/main/java/com/krech/botv3/service/FileService.java

```diff
@@ -0,0 +1,66 @@
+package com.krech.botv3.service;
+
+
+import com.krech.botv3.domain.WordObject;
+import com.krech.botv3.repository.WordRepository;
+
+
+import org.apache.tomcat.util.http.fileupload.disk.DiskFileItem;
+import org.apache.tomcat.websocket.Constants;
+import org.springframework.beans.factory.annotation.Autowired;
+
+
+import org.springframework.stereotype.Service;
+import org.springframework.web.multipart.MultipartFile;
+import org.springframework.web.multipart.commons.CommonsMultipartFile;
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
+
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardCopyOption;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+
+@Service
+public class FileService {
+
+
+    private final WordRepository wordRepository;
+
+    @Autowired
+    public FileService(WordRepository wordRepository) {
+        this.wordRepository = wordRepository;
+    }
+
+
+    public void readWordsFromFile(MultipartFile multipartFile) throws IOException {
+        List<String> list;
+
+        InputStream initialStream = multipartFile.getInputStream();
+        byte[] buffer = new byte[initialStream.available()];
+        initialStream.read(buffer);
+
+        File targetFile = new File("src/main/resources/targetFile.tmp");
```

Comment:
You should avoid creating file on server. It is better to read lines right from initialStream.